docarray/document/mixins/multimodal.py

```diff
@@ -139,3 +139,10 @@ def _from_obj(cls, obj, obj_type, field) -> typing.Tuple['Document', AttributeTy
         else:
             raise ValueError(f'Unsupported type annotation')
         return doc, attribute_type
+
+    def __getattr__(self, attr):
+        if self.is_multimodal and attr in self._metadata['multi_modal_schema']:
+            content = self.get_multi_modal_attribute(attr)[:, 'content']
```

Comment:
I am not sure that `content` is in all cases what we want, this is something i still have to check